pkg/cache/sql/informer/factory/informer_factory.go

```diff
@@ -20,17 +21,22 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+// EncryptAllEnvVar is set to "true" if users want all types' data blobs to be encrypted in SQLite
+// otherwise only variables in defaultEncryptedResourceTypes have their blobs encrypted
+const EncryptAllEnvVar = "CATTLE_ENCRYPT_CACHE_ALL"
+
 // CacheFactory builds Informer instances and keeps a cache of instances it created
 type CacheFactory struct {
-	informerCreateLock sync.RWMutex
-	wg                 wait.Group
-	dbClient           DBClient
-	stopCh             chan struct{}
-	encryptAll         bool
+	wg          wait.Group
+	dbClient    DBClient
+	stopCh      chan struct{}
+	stopChMutex sync.RWMutex
```

Comment:
nit: That mutex doesn't only guard the stopCh channel but also other fields like informers/informerMutexes. Seems like the name should be changed but I'm out of inspiration. Maybe just `lock`, with a comment saying what it is used for? WDYT?

Reply:
Fair point.

I like `mutex` better so I'll go down that path unless an illumination comes in the form of a much better name. Thanks

Reply:
fixed in 81eb208